<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/clearS.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="ipcon">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>64</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ip.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="portcon">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>104</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/duan.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_edi">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_edi">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>153</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>40</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>网络配置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>模式选择</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="wifi_l">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>25</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/wifi_off.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>17</y>
       <width>3</width>
       <height>55</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="sermode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/server.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="sermode_l">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>主机模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="connect_l">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/discon.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>          淡水鱼投食监控面板</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>余料检测部分</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="temp_la">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>77</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="humi_la">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>77</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>34</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/light.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>155</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>110</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/rain.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/mq2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="light_la">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>77</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="soil_la">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>153</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="mq2_la">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>153</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="rain_la">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>153</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/humi.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/temp.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/humi_soil.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="time_l">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>方正舒体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12：00：00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="charts1_big">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/big.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="charts1_small">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="charts1_rest">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rest.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>221</width>
      <height>167</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>投食设定</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="soil_yu_la">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rain_yu_la">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>58</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="temp_yu_la">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>88</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>33</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>g/次</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>61</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>h</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>89</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_yu_bt">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>138</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/set.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_yu_bt">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>137</y>
       <width>38</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/clear_yu.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>83</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>投喂间隔</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>83</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>定时(h)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>88</y>
       <width>83</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>定时(min)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>118</y>
       <width>83</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>定量</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>119</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>g</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="light_yu_la">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>118</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>QQVGA(160x120)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HQVGA(240x176)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QVGA(320x240)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CIF(400x296)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VGA(640x480)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SVGA(800x600)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XGA(1024x768)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SXGA(1280x1024)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UXGA(1600x1200)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>质量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>63</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>192.168.31.205</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>开始推流</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_13</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>graphicsView</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>ipcon</zorder>
   <zorder>portcon</zorder>
   <zorder>ip_edi</zorder>
   <zorder>port_edi</zorder>
   <zorder>line</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>time_l</zorder>
   <zorder>charts1_big</zorder>
   <zorder>charts1_small</zorder>
   <zorder>charts1_rest</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_wifi"/>
   <addaction name="separator"/>
   <addaction name="led"/>
   <addaction name="relay"/>
   <addaction name="auto_hand"/>
   <addaction name="separator"/>
   <addaction name="data"/>
   <addaction name="debb"/>
   <addaction name="separator"/>
   <addaction name="exit"/>
  </widget>
  <action name="exit">
   <property name="icon">
    <iconset resource="../img/picture.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="open_wifi">
   <property name="icon">
    <iconset resource="../img/picture.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>NetSw</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="led">
   <property name="icon">
    <iconset resource="../img/picture.qrc">
     <normaloff>:/led_off.png</normaloff>:/led_off.png</iconset>
   </property>
   <property name="text">
    <string>Led</string>
   </property>
  </action>
  <action name="relay">
   <property name="icon">
    <iconset resource="../img/picture.qrc">
     <normaloff>:/relay_off.png</normaloff>:/relay_off.png</iconset>
   </property>
   <property name="text">
    <string>Relay</string>
   </property>
  </action>
  <action name="auto_hand">
   <property name="icon">
    <iconset resource="../img/picture.qrc">
     <normaloff>:/auto.png</normaloff>:/auto.png</iconset>
   </property>
   <property name="text">
    <string>RunMode</string>
   </property>
  </action>
  <action name="debb">
   <property name="icon">
    <iconset resource="../img/picture.qrc">
     <normaloff>:/app.png</normaloff>:/app.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="data">
   <property name="icon">
    <iconset resource="../img/picture.qrc">
     <normaloff>:/data.png</normaloff>:/data.png</iconset>
   </property>
   <property name="text">
    <string>data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../img/picture.qrc"/>
  <include location="../img/picture.qrc"/>
  <include location="../img/picture.qrc"/>
 </resources>
 <connections/>
</ui>
